rotkehlchen/api/rest.py

```diff
@@ -2178,19 +2195,19 @@ def _import_data(
     def import_data(
             self,
             source: IMPORTABLE_LOCATIONS,
-            filepath: Path,
+            file: Union[FileStorage, Path],
```

Comment:
We need Union here since this function can be called both from both PUT and POST which pass different arguments

Reply:
Do not change the name from file, to filepath. There is a reason it was called `filepath`. `file` is a reserved keyword in python.

Reply:
Rolled back